modules/operator/variables.tf

```diff
@@ -0,0 +1,122 @@
+variable "name_prefix" {
+  description = "Prefix for all resource names (replaces separate namespace and environment variables)"
+  type        = string
+}
+
+variable "operator_version" {
+  description = "Version of the Materialize operator to install"
+  type        = string
+  default     = "v25.1.12" # META: helm-chart version
+  nullable    = false
+}
+
+variable "orchestratord_version" {
+  description = "Version of the Materialize orchestrator to install"
+  type        = string
+  default     = null
+}
+
+variable "helm_repository" {
+  description = "Repository URL for the Materialize operator Helm chart. Leave empty if using local chart."
+  type        = string
+  default     = "https://materializeinc.github.io/materialize/"
```

Comment:
Double checking: 

- With the new change, once this change is in, we'll want to, as part 2, update the self-managed release thing that Dennis showed to bump up this operator_version as well as no longer have the terraform-helm release bump trigger terraform-aws  bumps, yes?  

- And, is the plan that whenever the other cloud terraforms are updated, will we, at that time, remove terraform-helm-materialize? (or, we haven't quite decided upon that yet?)

Reply:
Yep! This is exactly right!

- We'll need to update Dennis’s workflow to align with the new restructured repo

- Also the CI tests should be updated to use the latest operator version

- Might be good to keep the old version on a separate `lts` branch in case some users still need it

- We'll most likely remove or deprecate the `terraform-helm-materialize` module, since we want to get rid of the Materialize instances from it, and without those, the module isn't very useful

- We've got this in the design doc here: https://github.com/MaterializeInc/cloud/pull/11040